modules/commons/src/test/java/org/apache/synapse/commons/json/JsonValueTest.java

```diff
@@ -119,4 +120,13 @@ public void runFailTest(String value) {
             assert true;
         }
     }
+
+    public void testWriteAsJsonNullElement() {
+        try {
+            OutputStream out = Util.newOutputStream();
+            JsonUtil.writeAsJson((OMElement) null, out);
```

Comment:
there should be a Assert.fail() statement to identify if the exception not thrown

Reply:
Done